packages/charts/src/components/CarbonChart/CarbonChart.tsx

```diff
@@ -0,0 +1,215 @@
+  }
+
+  .cds--btn {
+    min-height: auto;
+    display: flex;
+    flex: 0 auto;
+    align-items: center;
+    text-align: center;
+    padding: 20px;
+    margin: 0;
+    line-height:  ${props => props.theme.typeScale.size03.lineHeight};
+    margin: 0;
+    min-width:  ${props => props.theme.spaceScale.spacing13};
+    overflow: hidden;
+    padding: ;
+    position: relative;
+    right: ${props => props.theme.spaceScale.spacing04};
+    text-align: center;
+    height: 40px;
+    font-family: ${props => props.theme.bodyFont};
+    font-size: ${props => props.theme.typeScale.size03.fontSize};
+    border-radius: ${props => props.theme.borderRadius};
+    font-weight: 500;
+  }
+
+  .cds--btn--primary {
+    background: ${props =>
+      props.isInverse
+        ? props.theme.colors.primary600
+        : props.theme.colors.primary}
+  }
+
+  :focus {
+    outline: 2px solid ${props =>
+      props.isInverse
+        ? props.theme.colors.focusInverse
+        : props.theme.colors.focus};
+    }
+    outline-offset: 0;
+  }
+
+  .cds--btn--primary {
+    &:focus {
+        outline: 2px solid ${props =>
+          props.isInverse
+            ? props.theme.colors.focusInverse
+            : props.theme.colors.focus};
+        }
+        outline-offset: 0;
+        border: none;
+        box-shadow: none;
+    }
+  }
+
+  .cds--modal-close {
+    &:focus {
+      outline: 2px solid ${props =>
+        props.isInverse
+          ? props.theme.colors.focusInverse
+          : props.theme.colors.focus};
+      }
+      outline-offset: 0;
+      border: none;
+    }
+  }
+
+.cds--modal-footer.cds--modal-footer {
+    display: flex;
+    align-items: center;
+}
+`;
+
+// Carbon themes (https://github.com/carbon-design-system/carbon-charts/blob/master/packages/core/src/interfaces/enums.ts#L12)
+enum ChartTheme {
+  WHITE = 'white',
+  G100 = 'g100',
+  G90 = 'g90',
+  G10 = 'g10',
+}
+
+interface ColorsObject {
+  [key: string]: string;
+}
+
+export const CarbonChart = React.forwardRef<HTMLDivElement, CarbonChartProps>(
+  (props, ref) => {
+    const {
+      children,
+      testId,
+      isInverse: isInverseProp,
+      type,
+      dataSet,
+      options,
+      ...rest
+    } = props;
+    const theme = React.useContext(ThemeContext);
+    const isInverse = useIsInverse(isInverseProp);
+    const allCharts = {
+      bar: SimpleBarChart,
+      lollipop: LollipopChart,
+      grouped: GroupedBarChart,
+    };
+
+    function buildColors() {
+      let scaleColorsObj: ColorsObject = {};
+
+      const allGroups = dataSet.map(item => {
+        return 'group' in item ? item['group'] : null;
+      });
+      const uniqueGroups = allGroups.filter(
+        (g, index) => allGroups.indexOf(g) === index
+      );
+
+      uniqueGroups.forEach((group, i) => {
+        if (isInverse) {
+          return (scaleColorsObj[group || 'null'] =
+            theme.chartColorsInverse[i]);
+        } else if (uniqueGroups.length <= theme.chartColors.length) {
```

Comment:
We need to check if `isInverse` is true, use `chartColorsInverse` else `chartColors`